bep001-naming-conventions.md

```diff
@@ -0,0 +1,73 @@
+Template:
```

Comment:
I'm not 100% sure this file should be in here...

Reply:
Yes, my mistake. I rebased it to Agah's work, but that is not appropriate here.